server/views/assessments/pages/matching-information/matching-information.njk

```diff
@@ -18,33 +18,110 @@
   {{
     formPageCheckboxes(
       {
-        fieldName: "mentalHealthSupport",
+        fieldName: "specialistSupportCriteria",
         fieldset: {
           legend: {
-            text: page.mentalHealthSupport.question,
+            text: "If this person would benefit from specialist support, select the relevant option below",
             classes: "govuk-fieldset__legend--m"
           }
         },
         hint: {
-          text: page.mentalHealthSupport.hint
+          html: "<p>There are only two AP with a semi-specialism in mental health nationally. Placement in one of these AP is not guaranteed.<p><p>Recovery Focused AP's are currently being piloted nationally. Placement is not guaranteed.</p>"
```

Comment:
minor: is there a missing closing p tag?
```suggestion
          html: "<p>There are only two AP with a semi-specialism in mental health nationally. Placement in one of these AP is not guaranteed.</p><p>Recovery Focused AP's are currently being piloted nationally. Placement is not guaranteed.</p>"
```

Reply:
Whoops! Have fixed 👍